lib/util.js

```diff
@@ -76,14 +76,23 @@ exports.generateStreamArrayForLevels = function (lvls, stream) {
  * @return {Array}
  */
 exports.getDefaultStreams = function () {
-  return (env.get('TKE_LOG_SILENT') === 'true') ? [] : [{
+  var logStreams = [{
```

Comment:
Why are we removing the SILENT handling? Isn't this useful for testing or disabling logs?